content/implementations.md

```diff
@@ -16,51 +16,52 @@ github:
+
 ## Test Methodologies
 
-Methodologies provide step by step instructions on how to test accessibility standards.
-See [understanding ACT consistency](#understandingactconsistency).
+Test methodologies provide step by step instructions on how to test accessibility requirements.
+For details, see [understanding ACT consistency](#understandingactconsistency).
 
 {% assign methodologies = implementations | where: "type", "Test methodology" %}
 {% include_relative _implementations-table.html implementations=methodologies %}
 
 ## Semi-automated Test Tools
 
-Tools that combine user input and automated testing to test accessibility standards.
-See [understanding ACT consistency](#understandingactconsistency).
+Semi-automated tools combine user input and automated testing to test accessibility requirements.
+For details, see [understanding ACT consistency](#understandingactconsistency).
 
 {% assign semiAutoTools = implementations | where: "type", "Semi-automated tool" %}
 {% include_relative _implementations-table.html implementations=semiAutoTools %}
 
-## Automatic Test Tools
+## Automated Test Tools
 
-Tools that fully automatically test accessibility standards.
-See [understanding ACT consistency](#understandingactconsistency).
+Automated test tools test accessibility requirements without user input.
+For details, see [understanding ACT consistency](#understandingactconsistency).
 
 {% assign automatedTools = implementations | where: "type", "Automated tool" %}
 {% include_relative _implementations-table.html implementations=automatedTools %}
 
 ## Understanding ACT Consistency
 
-Scores on this page are based on publicly available test results. Each ACT rule has examples. Vendors of test tools and methodologies run their implementation against the examples and report the outcome in a standard format. By comparing the outcome to the expected result, the consistency of these implementations can be worked out.
+Each Test Rule has examples, known as test cases. Evaluation tool vendors and test methodology developers run their implementations against the examples. Then they report the outcome in a standard format.
 
-A test tool or methodology is considered **fully consistent** when all the following are true:
+Consistency of an ACT implementation relates to how the outcomes of an implementation match the expected result of each of the Test Rules examples.
 
-- all success criteria that should fail are reported as such; and
-- all examples have the expected outcome. (I.e. a failed example should not be passed or vise versa.); and
-- there are outcomes for all examples; and
-- automated implementations may report a "cannot tell" on some, but not all examples and still be considered consistent.
+* An ACT implementation is **fully consistent** when all the following are true:
```

Comment:
I'm not convinced about putting these as nested lists. It sort of suggests that fully consistent and partially consistent are more or less equivalent to each other. I don't think that's the case. Implementations should be fully consistent. You can get partial credit for not getting things wrong, but the emphasis should be on getting things right. A partial implementation didn't get it all right.